cmd/mysterium_client/cli/command.go

```diff
@@ -147,7 +147,7 @@ func (c *Command) connect(line string) {
 }
 
 func (c *Command) unlock(line string) {
-	unlockArgs := strings.TrimSpace(line[7:])
+	unlockArgs := strings.TrimSpace(line[6:])
```

Comment:
There is quite a lot of boiler-plate code in cli - this error might have been avoided by having "unlock" string const and using i.e. `strings.TrimSpace(line[len(UNLOCK)])`. But that's beyond scope of this PR.